package.json

```
@@ -20,31 +20,31 @@
   "dependencies": {
     "@hapi/accept": "^5.0.1",
     "@hapi/ammo": "^5.0.1",
-    "@hapi/boom": "9.x.x",
+    "@hapi/boom": "^9.1.0",
     "@hapi/bounce": "2.x.x",
     "@hapi/call": "8.x.x",
-    "@hapi/catbox": "^11.1.1",
+    "@hapi/catbox": "^11.1.0",
     "@hapi/catbox-memory": "5.x.x",
-    "@hapi/heavy": "^7.0.1",
-    "@hapi/hoek": "9.x.x",
+    "@hapi/heavy": "7.x.x",
+    "@hapi/hoek": "^9.0.4",
     "@hapi/mimos": "5.x.x",
-    "@hapi/podium": "^4.1.1",
-    "@hapi/shot": "^5.0.1",
+    "@hapi/podium": "^4.1.0",
+    "@hapi/shot": "5.x.x",
     "@hapi/somever": "3.x.x",
-    "@hapi/statehood": "^7.0.3",
+    "@hapi/statehood": "^7.0.2",
     "@hapi/subtext": "^7.0.3",
-    "@hapi/teamwork": "5.x.x",
+    "@hapi/teamwork": "^5.1.0",
     "@hapi/topo": "5.x.x",
-    "@hapi/validate": "^1.1.0"
+    "@hapi/validate": "^1.1.1"
   },
   "devDependencies": {
     "@hapi/code": "8.x.x",
-    "@hapi/inert": "^6.0.2",
+    "@hapi/inert": "^6.0.1",
```

Let's leave this as is, as joi was dropped as a dependency from inert in 6.0.2.
```suggestion
    "@hapi/inert": "^6.0.2",
```